ibis/backends/duckdb/tests/conftest.py

```diff
@@ -14,3 +16,15 @@ def __init__(self, data_directory: Path) -> None:
     def connect(data_directory: Path):
         path = data_directory / "ibis_testing.ddb"
         return ibis.duckdb.connect(str(path))  # type: ignore
+
+
+@pytest.fixture(scope="session")
+def tpch(data_directory):
+    import duckdb
```

Comment:
In theory `duckdb` can be imported at the top level without causing annoying failures when it's not installed.